pyins/earth.py

```diff
@@ -19,12 +19,38 @@
 #: Gravity at the pole.
 GP = 9.8321849378
 
-F = R0 * (1 - E2) ** 0.5 * GP / (GE * R0) - 1
+F = (1 - E2) ** 0.5 * GP / GE - 1
 #: Standard gravitational parameter for Earth
 MU = 3.986004418e14
 
 
-def principal_radii(lat):
+def rate_n(lat):
```

Comment:
Is it used somewhere?

Reply:
Doesn't look it is used anywhere.

Reply:
You're  right, initially I used it in integrator. I will prefer to keep it for future use if you are not strongly against it.